packages/cozy-harvest-lib/src/components/KonnectorConfiguration/ConfigurationTab/bankAccountHelpers.js

```diff
@@ -0,0 +1,74 @@
+import { get, uniqBy, flatten } from 'lodash'
+
+const PARTS_TO_DELETE = ['(sans Secure Key)']
```

Comment:
is it from BI?

Reply:
No, it is a rule that we had in Banks that I haven't touched. I think it is directly from the bank.